### PR TITLE
Servlet Setup

### PR DESCRIPTION
**Pull request by elenapelaez**
Steps 1 and 2 for Week 2. I setup a servlet and used JavaScript to get a response from the Java servlet. I used the Cloud Shell Preview tool to make sure the JavaScript was grabbing the server's response correctly.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <script src="script2.js"></script>
```
Comment (spunja-sps) [inline portfolio/src/main/webapp/index.html]: Could you rename the js file to something more descriptive?

File: portfolio/src/main/webapp/script2.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+async function getMessage() {
```
Comment (spunja-sps) [inline portfolio/src/main/webapp/script2.js]: Could you add a comment here explaining what the getMessage method does.

Eg: Fetches response from /hello and replaces text in the "greeting-container"

**Comment by spunja-sps** [comment]
Can you please update the PR title to something more descriptive.

File: portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java [hunk 3]
```diff
@@ -5,14 +5,25 @@
...
+    positiveMessages.add("Keep your face to the sunshine and you cannot see a shadow");
+    positiveMessages.add("The only time you fail is when you fall down and stay down");
+    positiveMessages.add("If opportunity doesn't knock, build a door");
+    positiveMessages.add("Live life to the fullest and focus on the positive");
+
+    Gson gson = new Gson();
+    String json = gson.toJson(positiveMessages);
+
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
```
Comment (rajeshmalireddy) [inline portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java:27]: Can we make these class constants.

You can try using guava ImmutableList and build

```
private static final List<String> messages =
    ImmutableList.builder()
        .add("message-1")
        .add("message-2")
        .build();
```

or 

```
private static final List <String> messages = Arrays.asList("messsage-1", "message-2",....);
```

```
private static final String message_as_json = new Gson().toJson(messages);
```

**Comment by elenapelaez** [comment]
I updated my code for all of the given comments

**Pull request merged**